main.js

```diff
@@ -1 +1,35 @@
 import customers from './customers.js'
+
+//you will need their name, address, phone numbers, thumnail, birthdate, and the date they became a customer
+
+let person = [{
+ // thumbNail:'https://randomuser.me/api/portraits/thumb/women/89.jpg',
+  name: "pauline hughes",
+ // email: "pauline.hughes@example.com",
+ // address: "3447 Elgin St, Roseville, IN 93281",
+  // birthdate: "12/08/1977" ,
+  // startDay: "06/29/2004"
+}]
+
+//person.push({
+ // thumbNail:'https://randomuser.me/api/portraits/thumb/women/89.jpg',
+ // name: "Pauline Hughes",
+ // email: "pauline.hughes@example.com",
+//  address: "3447 Elgin St, Roseville, IN 93281",
+//})
+
+function listCustomers(personArray) {
+  let customerList = document.querySelector("#customer-list");
```

Comment:
Nice work. A next step is to make this function handle any array you input, so to do this loop on `inputArray` instead of specifically `customers`.